cargo-dist/src/errors.rs

```diff
@@ -171,7 +171,7 @@ pub enum DistError {
     },
     /// Not an error; indicates that a file's contents differ via --check
     #[error("{} has out of date contents and needs to be regenerated:\n{diff}", file.origin_path())]
-    #[diagnostic(help("run 'cargo dist init' to update the file or set 'allow-dirty' in Cargo.toml to ignore out of date contents"))]
+    #[diagnostic(help("run 'cargo dist init' to update the file\n( 'allow-dirty' in Cargo.toml to ignore out of date contents)"))]
```

Comment:
No space after opening paren, IMO

```suggestion
    #[diagnostic(help("run 'cargo dist init' to update the file\n('allow-dirty' in Cargo.toml to ignore out of date contents)"))]
```